api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -824,6 +824,15 @@ private void checkGetPersonNameResultForVoidedPerson(PersonName expectedPersonAd
 		
 		assertEquals(expectedPersonAddress, actualPersonName);
 	}
+
+	@Test
+	public void shouldSetDeadToTrueIfSetDeathdate() {
+
+				Person p = new Person();
+				Date deathDate = new Date();
+				p.setDeathDate(deathDate);
+				assert (p.getDead());
```

Comment:
What you removed is what we use else where. Have you looked at other existing tests to confirm what am saying.
Can you use assertTrue as here? https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/PersonTest.java#L58